frontends/concrete-python/examples/pir/pir_utils.py

```diff
@@ -0,0 +1,55 @@
+import numpy as np
+
+from concrete import fhe
+
+
+@fhe.compiler({"one_hot_vector": "encrypted", "database": "clear"})
+def get_ith_element_of_database(one_hot_vector: np.ndarray, database: np.ndarray) -> int:
+    return np.dot(one_hot_vector, database)
+
+
+def make_one_hot_vector(index: int, size: int) -> np.ndarray:
+
+    answer = np.zeros(shape=(size,), dtype=np.int8)
```

Comment:
is that dtype is madatory?

Reply:
It's more query than answer, right?

Reply:
- if we don't have dtype, i think it takes float, no?
- "It's more query than answer, right?": I have not understood

Reply:
removing this file